aspnetcore/fundamentals/routing/samples/8.x/ShortCircuitSample/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Debug",
+      "Microsoft.Hosting.Lifetime": "Information"
```

Comment:
```suggestion
      "Microsoft": "Debug"
      ,"Microsoft.Hosting.Lifetime": "Information"
```

Reply:
Looks funny but is correct. The editor will fix it up for the customer.

Reply:
I changed the instructions to say "set Microsoft to Info", so that addresses the situation where they already have the line or don't have it.